docs/lp-mock/beutl-lp-full.html

```
@@ -0,0 +1,736 @@
+<title>Beutl LP — 全体モック</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)